git-hg/mercurialToGit.sh

```diff
@@ -54,6 +54,32 @@ HG_REPO=$1
 GITHUB_REPO=$(echo "$HG_REPO" | cut -d/ -f2)
 BRANCH=${2:-master}
 
+# Determine latest build TAG_SEARCH, finding tags beginning with "jdk-" followed by ... "+" ...
+if [ "$HG_REPO" == "jdk/jdk" ]
```

Comment:
Will this be brittle?  In that, we have to update this every 6 months to add new repos.  If so that's OK but we should but that in the README

Reply:
I'd see that as part of the adding of the related Jenkins job work... I could parse the HG_REPO, but I wanted to make sure we're finding exactly the right tags and some though is put into each new repo that gets added...